CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 # [Versions](https://mui.com/versions/)
 
+## 7.0.0-beta.5
+
+<!-- generated comparing v7.0.0-beta.3..master -->
+
+_Mar 18, 2025_
+
+A big thanks to the 10 contributors who made this release possible.
+
+### `@mui/material@7.0.0-beta.5`
+
+- [Accordion] Add missing `root` slot (#45532) @sai6855
+- [AccordionSummary] Add slots and slotProps (#45559) @sai6855
+- [ListItemText] Add missing `root` slot (#45540) @sai6855
+- [SpeedDial] Add missing `root` slot (#45541) @sai6855
+- [StepLabel] Add missing root slot (#45603) @sai6855
+- [Switch] Add slots and slotProps (#45595) @siriwatknp
+- [Tooltip] Allow auto placement on tooltip (#45399) @Jtaks
+- [TextareaAutosize] Fix ResizeObserver causing infinite `selectionchange` loop (#45351) @mj12albert
+- [useScrollTrigger] Do nothing if target is null (#45441) @vipierozan99
+- [core] Rename some `./styles/createTypography` exports (#45558) @Janpot
+- [utils] use correct iri-reference homepage format [NOTICKET] (#45472) @dahiro
+- Add merging function to `mergeSlotProps` utility (#45543) @siriwatknp
+- Fix negative input for CSS variables spacing array (#45550) @siriwatknp
+- Add missing deprecations in deprecations-all file (#45505) @sai6855
```

Comment:
Many of these PRs were already released in [`7.0.0-beta.4`](https://github.com/mui/material-ui/releases/tag/v7.0.0-beta.4). Same in other sections below.

Reply:
Oh, let me check.

Reply:
Updated